esx_service/utils/auth_api.py

```diff
@@ -554,6 +579,17 @@ def _tenant_access_add(name, datastore, allow_create=None, default_datastore=Fal
         error_info = error_code.generate_error_info(ErrorCode.PRIVILEGE_ALREADY_EXIST, name, datastore)
         return error_info
 
+    if allow_create is not None:
```

Comment:
nit: please add comment '# note - the value can be None (no changes), or True/False ("change to this value)")